src/python/zquantum/core/circuits/__init__.py

```diff
@@ -141,7 +141,9 @@
     builtin_gate_by_name,
 )
 from ._circuit import Circuit
+from ._cirq_decompositions import PowerGateToPhaseAndRotation, decompose_cirq_circuit
 from ._compatibility import new_circuit_from_old_circuit
+from ._decomposition import DecompositionRule, decompose_operations
```

Comment:
The brand new decomposition engine seems to be unrelated to our Z-Quantum circuits framework. WDYT about moving it to a different place in the project, e.g. `zquantum.core.decomposition`?

Reply:
Moved to `zquantum.core.decomposition` in 36913a9